lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -299,6 +307,24 @@
   stampTemplate('unoptimized-images-tmpl', document.body);
 }
 
+function deprecationsTest() {
+  stampTemplate('deprecations-tmpl', document.head);
+
+  // Some APIs atm are not triggering Chrome's deprecation messags. https://crbug.com/680832.
+  // try {
+  //   MediaStreamTrack.getSources(); // FAIL
+  // } catch(e) {}
+
+  // console.timeline(); // FAIL
```

Comment:
I don't think we want to do that b/c smokehouse tests will fail indefinitely until Chrome fixes it. That could be an annoying while :\